bootstrap/manifest_template.json

```diff
@@ -125,12 +125,72 @@
     }
   },
   {
+    "id": "redis",
+    "app": {
+      "name": "redis",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "action": "run-app",
```

Comment:
I cleaned up the manifest in 478e3a3. The `redis` steps weren't needed since they're done by `redis-api`.